spec/fixtures/create_multiple_ini_settings/manifests/init.pp

```diff
@@ -9,12 +9,11 @@
   $example = {
     'section1' => {
       'setting1'  => 'value1',
-      'settings2' => {
+      'setting2' => {
         'ensure' => 'absent'
       }
     }
   }
 
-  create_ini_settings($example, $defaults)
+  inifile::create_ini_settings($example, $defaults)
```

Comment:
This test is updated to use the namespaced version, but I've left `spec/fixtures/create_ini_settings_test/manifests/init.pp` as-is to use the non-namespaced shim.